prow/tide/tide.go

```diff
@@ -1194,6 +1218,46 @@ func (pr *PullRequest) logFields() logrus.Fields {
 	}
 }
 
+func (pr *PullRequest) approverTags(tag string) string {
+	if len(pr.Reviews.Nodes) == 0 {
+		return ""
+	}
+
+	approverSet := map[string]bool{}
+	for _, review := range pr.Reviews.Nodes {
+		approverSet[review.Author.String()] = true
+	}
+
+	approvers := make([]string, len(approverSet))
+	i := 0
+	for approver := range approverSet {
+		approvers[i] = approver
+		i += 1
```

Comment:
A more idiomatic way to do this would be to start with `approvers := make([]string, 0, len(approverSet))` which sets the length to 0, but the capacity to the appropriate value. Then you can efficiently use `approvers = append(approvers, approver)` inside the loop.

Alternatively make `approverSet` a `sets.String` and then use `approverSet.List()` to return a sorted list of the set values.